app.py

```diff
@@ -147,6 +163,58 @@ def graph_refresh() -> None:
             return
 
 
+def graph_refresh_package_analyzer() -> None:
+    """Schedule refresh for packages that are not yet analyzed by package analyzer."""
+    graph = GraphDatabase()
+    graph.connect()
+
+    packages = graph.retrieve_unanalyzed_python_package_versions()
+
+    if not packages:
+        _LOGGER.info("No unanalyzed packages found")
+        return
+
+    count = 0
+    for item in packages:
+        package = item["package_name"]
+        version = item["package_version"]
+        url = item["index_url"]
+        try:
+            analysis_id = openshift.schedule_package_analyzer(
+                package_name=package,
+                package_version=version,
+                index_url=url,
+                output=_PACKAGE_ANALYZER_OUTPUT,
+            )
+        except Exception as ecx:
+            # If we get some errors from OpenShift master - do not retry. Rather schedule the remaining
+            # ones and try to schedule the given package in the next run.
+            _LOGGER.exception(
+                f"Failed to schedule new package analyzer to analyzer package {package} in version {version}"
+                f"from {url}, the graph refresh job will not fail but will try to reschedule this in next run"
```

Comment:
Is it OK to concatenate like these? I mean I have two `f`s in two different lines.